### PR TITLE
reduce boilerplate code

### PR DESCRIPTION
**Pull request by svenwltr**
> Based on https://github.com/rebuy-de/rebuy-go-sdk/pull/2

Todo:

* [x] change sdk version after merging https://github.com/rebuy-de/rebuy-go-sdk/pull/2

### CODE REVIEW

**Comment by svenwltr** [comment]
@rebuy-de/it-platform Please review.

**Comment by svenwltr** [comment]
Fixed `go vet` since it needs the vendor directory with Go 1.10.

File: example/golang.mk [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Source: https://github.com/rebuy-de/golang-template
 # Version: 1.4.1-snapshot
 # Dependencies:
-# * Godep
```
Comment (bethge) [inline example/golang.mk]: \o/

**Comment by svenwltr** [comment]
Also added `-s` for `gofmt` to enforce simplified code.

**Pull request merged**